core/kazoo_voicemail/src/kvm_message.erl

```diff
@@ -337,6 +354,10 @@ do_copy(AccountId, FromId, OldBoxId, NBId, NBoxJ) ->
                          ,[FromDb, FromId, ToDb, ToId]
                          ),
             NotFound;
+        {'error', 'timeout'} ->
+            do_copy(AccountId, FromId, OldBoxId, NBId, NBoxJ, Loop - 1);
+        {'error', 'conflict'} ->
```

Comment:
What would cause a conflict in this situation ?  Also what would resolve it ? Unless it is something external to this process retrying the same operation would never succeed, correct ?